### DISCUSSION TITLE
Assistance Needed for GitHub Pages Site Accessibility Issue

### DISCUSSION THREAD

**Discussion by oajoku** [community/community#140695] [New to GitHub]
Hello,

I hope this message finds you well. My name is Onyinyechi Ajoku, and I’m a beginner in web development. I’ve been trying to create a portfolio website and host it using GitHub Pages, but I am encountering an issue where the site isn’t accessible on search engines.

Here are the steps I've taken so far:

Repository Setup: I created a repository called oajoku.github.io and set it up for GitHub Pages.
File Structure: I have added my portfolio files to a /docs folder within the repository, and I also moved the necessary Google verification file to the root of the repository.
Publishing Source: I created a GitHub repository and configured the publishing source to use the "master" branch, selecting the /docs folder.
Verification File: I added the Google verification HTML file as instructed, but Google Search Console is still unable to find it, and I am seeing a "404 Not Found" error when trying to access my site via the link https://oajoku.github.io/.
Despite following various guides and troubleshooting steps, I haven’t been able to resolve this issue. I would greatly appreciate any help or guidance you can provide on how to ensure my portfolio site is properly set up and accessible online.

Thank you for your time and support!

Best regards,
Onyinyechi Ajoku

**Comment by Topromm** [comment]
**Check the Repository Name**: Make sure the repo name is exactly oajoku.github.io with no extra spaces or characters.

**File Placement**: Since you’ve selected the /docs folder as the publishing source, double-check that your HTML, CSS, and other files needed for your site are inside the /docs folder, and not in any subfolders. Also, make sure there's an index.html file.

**Google Verification File**: If you're seeing a 404 error, the verification file might not be where Google expects it. Since you have your site in the /docs folder, make sure the verification file is either in /docs (if that’s your root), or try placing it directly in the repository root to see if that resolves the issue.

**GitHub Pages Settings**: Double-check that the publishing source is correctly set to the master branch and /docs folder in the repository's settings (under the "Pages" section). After making changes, sometimes you have to wait a few minutes for them to take effect.

If you’re still getting the 404 error when trying to access https://oajoku.github.io/, it could be that the site hasn’t been fully published yet. You can force a rebuild by making a minor change (like editing the README.md or your HTML file) and committing again.

Once your site is accessible, it might still take a while for Google to index it. You can manually request indexing via Google Search Console to speed it up if wanted.

If the 404 persists, try accessing a specific file in the /docs folder directly (like https://oajoku.github.io/docs/index.html) to see if it's available there.

Hope this helps!

**Reply by oajoku** [reply]
Thank you so much for responding ! I eventually deleted the repository after figuring out I had made a mistake in the HTML code but I'll try again with these suggestions!

**Reply by ghostinhershell** [reply]
Hi @oajoku 👋🏾,It looks like @Topromm 's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)
